From 8f557a737e7f7973e2186e1b77ddb966e7086373 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Wed, 1 Jun 2011 09:31:04 +0800
Subject: [PATCH 223/505] ENGR00143958 ar3k_bt: change file path as absolute
 path

change hciattach-ar3k.bin file path in the script file
as absolute path.

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 test/ar3k_bt/ar3k-bt-start.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/ar3k_bt/ar3k-bt-start.sh b/test/ar3k_bt/ar3k-bt-start.sh
index 840932f..82e0c3a 100644
--- a/test/ar3k_bt/ar3k-bt-start.sh
+++ b/test/ar3k_bt/ar3k-bt-start.sh
@@ -3,4 +3,4 @@
 sudo pkill -9 hciattach-ar3k.bin
 sudo su -c "echo 1 > /sys/class/rfkill/rfkill0/state"
 sudo chmod 777 /dev/ttymxc2
-sudo ./hciattach-ar3k.bin -n ttymxc2 ath3k 3000000 flow sleep &
+sudo /unit_tests/hciattach-ar3k.bin -n ttymxc2 ath3k 3000000 flow sleep &
-- 
1.8.0

